### PR TITLE
Add Lockhunter

### PR DESCRIPTION
**Pull request by vincecima**

### CODE REVIEW

File: just-install.json [hunk 1]
```diff
@@ -398,6 +398,14 @@
       },
       "version": "5.0.4"
     },
+    "lockhunter": {
+      "installer": {
+        "kind": "innosetup",
+        "x86": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe",
+        "x86_64": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe"
```
Comment (lvillani) [inline just-install.json]: There's no need to provide the x86_64 entry. just-install automatically falls-back to x86 if it's the only one available, even on 64-bit systems. Add the x86_64 entry only if there's an actual 64-bit installer.

Comment (vincecima) [inline just-install.json]: Lockhunter ships an installer that picks the correct version to install (32 or 64). In that case, should we have entries for both `x86` and `x86_64`?

Comment (lvillani) [inline just-install.json]: ConEmu actually does something similar. I prefer to add both entries only when URLs are different.

Comment (vincecima) [inline just-install.json]: :+1: will change. For situations like this, any value in adding a `all` or `both` arch? I realize those make less sense if desktop Windows gets ported to ARM or something like that.

**Comment by vincecima** [comment]
@lvillani all set

**Comment by lvillani** [comment]
Awesome! 

cb21eda885b4cc6ee3f549a417770596

**Pull request merged**